cabal.project

```diff
@@ -10,3 +10,6 @@ packages:
   codeworld-server/
   codeworld-base/
   funblocks-client/
+
+constraints:
+  ghcjs-dom ==0.9.5.0
```

Comment:
This should really go in the cabal file (presumably in `codeworld-api/codeworld-api.cabal`) rather than here.  To be honest, I'm a little surprised that there's a cabal.project file checked in.  That seems wrong...

Reply:
Regarding the `cabal.project` file, I know the primary way to build CodeWorld is with the scripts provided; needed for GHC patches, etc. IMO the `cabal.project` file is useful nonetheless. For one, I'm trying to get the whole project building with Nix https://github.com/google/codeworld/pull/1523 and `cabal.project` is needed for that.